### PR TITLE
Update text on Elevate widget in template builder

### PR DESCRIPTION
**Pull request by dfuller114**
# Critical Changes

# Changes
Admins will see text in the elevate widget card in template builder notifying them that they should add the payment method field to the template if it has not already been added.

# Issues Closed

# Community Ideas Delivered

# New Metadata

# Deleted Metadata

### CODE REVIEW

File: src/lwc/geTemplateBuilderFormField/geTemplateBuilderFormField.js [hunk 1]
```diff
@@ -106,6 +106,14 @@ export default class geTemplateBuilderFormField extends LightningElement {
         }
     }
 
+    get renderedWidgetClass() {
```
Comment (psadie-sfdo) [inline src/lwc/geTemplateBuilderFormField/geTemplateBuilderFormField.js:109]:
```suggestion
    get cssClassRenderedWidget() {
```

Minor suggestion to include `css` somewhere in the name. Quickly clear that this is solely for css when doing a quick search for 'class' keyword.

**Review by psadie-sfdo — COMMENTED** [review]
Just one minor comment. Let me know if you agree/disagree and I'll approve shortly!

**Comment by dfuller114** [comment]
work item: https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07B0000008jPylIAE/view

**Review by psadie-sfdo — APPROVED** [review]
Looks good!

**Pull request merged**